pebbles/static/js/controllers/BlueprintsController.js

```diff
@@ -227,6 +227,14 @@ app.controller('ModalCreateBlueprintController', function($scope, $modalInstance
 
 app.controller('ModalReconfigureBlueprintController', function($scope, $modalInstance, blueprint, groups_list) {
     $scope.blueprint = blueprint;
+    var config_mismatch = false;
+    blueprint.get().then(function(response){
+        if(!_.isEqual($scope.blueprint['config'], response['config'])){  // Object equality check
+            alert('Blueprint configuration has been changed from the previous configuration');
+            $scope.blueprint = response;
+            config_mismatch = true; //Concurrency
+        }
+    });
     //$scope.groups = groups_list;
     //$scope.groupModel =  _.filter(groups_list, {'id': blueprint.group_id})[0];
     //$scope.updateBlueprint = function(form, model, groupModel) {
```

Comment:
I left the whole block here if future there's a need to change the group of an existing blueprint (which we decided should not be needed as of now)